packages/winpanda/extra/src/winpanda/core/cmdconf.py

```diff
@@ -454,6 +456,7 @@ def get_dstor_dcoscfgpkg_path(self, dstor_root_url: str,
                 f' {e}'
             ) from e
 
+        lpi_fname = Path(dstor_lpi_path).name
```

Comment:
This change will conflict with my most recent change. In `master` this line is no longer needed because we use the path returned by `cm_utl.download`

Reply:
okay will do rebase and rollback this line after